docs/source/maintainers.rst

```diff
@@ -0,0 +1,63 @@
+=====================
+Maintainer Guidelines
+=====================
+
+This page describes best practices for Dask maintainers.
+
+Thank you for helping make Dask a useful library and fostering a
+welcoming community.
+
+Merging pull requests
+=====================
+
+Pull requests should be reviewed
+--------------------------------
+
+Pull requests from non-maintainers should be reviewed and approved by at
+least one maintainer before being merged.
+
+Ideally, pull requests from maintainers should also be reviewed before
+merging. However, because reviewer bandiwdth is limited, mainainers will
+sometimes self-merge their own pull requests if the contents of the pull
+request is viewed as uncontroversial (e.g. typo fix). If a maintainer has
+a substantial pull request which hasn't received a review, and the maintainer
+is confident in the proposed changes, they should post a final comment along
+the lines of "Merging in 24 hours if no further response" and then self-merge
+if no further comments are left within the specified time period.
+
+No maintainer should merge a pull request that they're not comfortable with.
```

Comment:
oh interesting! I read this as "don't merge if you don't understand the new code"

Reply:
@mrocklin made a point at SciPy that the person who hits merge should be available to handle any fallout in the short term. That's what I'm trying to capture here but may not be putting it into the right words.

Reply:
What I meant to say is that as we merge things we should have a plan for if they break and cause issues.  That plan could be colleagues or some other arrangement.  We probably shouldn't be merging things if we don't have such a plan though.

Reply:
Oh yeah I get what you all are saying. I was just trying to agree that the current language is unclear, because I understood it differently from you. I think your suggestion is good.

Reply:
Made this update -- thanks!